boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -32,12 +32,6 @@ static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
 
         const bool is_brake_actuated = App_CanRx_FSM_Brake_IsActuated_Get();
 
-        // TODO: JSONCAN
-        // #ifndef NDEBUG
-        //             || App_CanRx_CAN_DEBUGGING_SIGNALS_GetSignal_FORCE_DCM_TO_DRIVE_FROM_INIT(can_rx_interface)
-        // #endif
-        //         )
-
```

Comment:
Just checking, did this get addressed?

Reply:
Let me check with Joe what he wants for this
Its really not that hard to get into drive state, maybe we don't need this